MatrixSDK/Crypto/CryptoMachine/MXCryptoMachine.swift

```diff
@@ -117,7 +117,8 @@ class MXCryptoMachine {
     }
     
     func invalidateCache() async {
-        await machine.clearCryptoCache()
+        // Not sure what is the new function for this
+//        await machine.clearCryptoCache()
```

Comment:
this function seems to not exist anymore in this crypto sdk version, what is the replacement?
@poljar

Reply:
The cache doesn't exist anymore, at least not inside the store. Rotating the Olm machine would clear all caches now.